frontend/package.json

```diff
@@ -14,6 +14,7 @@
     "@use-it/interval": "^1.0.0",
     "draft-js": "^0.11.7",
     "draftjs-to-html": "^0.9.1",
+    "follow-redirects": "^1.14.7",
```

Comment:
Do we need an entry here for `follow-redirects` if we add one to the `resolutions` section below?

Reply:
yarn upgrade must have added this automatically, I'll remove it